application/client/src/app/ui/views/toolbar/search/input/component.ts

```diff
@@ -111,6 +123,10 @@ export class ViewSearchInput
         this.input.actions.recent.subscribe(() => {
             this.markChangesForCheck();
         });
+        this.input.actions.reaccept.subscribe(() => {
+            this._updateSearch(undefined);
+        });
```

Comment:
in general with `accept` handler and `reaccept` happens same things - we are triggering a search. Do we need extra-event `reaccept` then?